packages/compass-workspaces/src/stores/workspaces.ts

```diff
@@ -361,30 +366,10 @@ const reducer: Reducer<WorkspacesState> = (
       WorkspacesActions.CollectionRenamed
     )
   ) {
-    let tabsRenamed = 0;
-    let newActiveTabId = state.activeTabId;
-    const newTabs = state.tabs.map((tab) => {
-      if (tab.type === 'Collection' && tab.namespace === action.from) {
-        tabsRenamed++;
-        const { id, ...workspace } = tab;
-        const newTab = getInitialTabState({
-          ...workspace,
-          namespace: action.to,
-        });
-        if (id === state.activeTabId) {
-          newActiveTabId = newTab.id;
-        }
-        return newTab;
-      }
-      return tab;
-    });
-    if (tabsRenamed === 0) {
-      return state;
-    }
```

Comment:
Why is this removed from there? Reducer shoud be deriving state from previous state and actions as per [redux guidelines](https://redux.js.org/style-guide#put-as-much-logic-as-possible-in-reducers)